src/actions/TransactionActions.js

```diff
@@ -75,6 +75,7 @@ import {
 	validateAmount,
```

Comment:
Reset label when input is empty, contract id or something you can't get an account name or id (eth, btc addresses)